TimeTrackerApp/TimeTrackerApp/Utilites/Extensions/Color+Extension.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Color+Extension.swift
+//  TimeTrackerApp
+//
+//  Created by Sevda Gul Baran on 13.09.2022.
+//
+
+import Foundation
+import UIKit
+
+
+import UIKit
+
+extension UIColor {
+
+    class var pink: UIColor {
+        let const = 0xFD5B71
+        return UIColor.rgb(fromHex: const)
```

Comment:
renk kullanımları güzel, tek satırda yapabilirsin. Swift 5 ile return'e gerek yok tek satırda. class variable kullandığın zaman inheritance yapıldığında override edilme yetkisi veriliyor. isteyerek yaptıysan ok ama genelde biz bunu istemiyoruz ve static var yapıyoruz.



```suggestion
    static var pink: UIColor {
        .rgb(fromHex: 0xFD5B71)
```